portal-web/test/functional/com/liferay/portalweb/MyDMTests/MyDMTests.testcase

```diff
@@ -0,0 +1,91 @@
+@component-name = "portal-document-management"
+definition {
+    property testray.main.component.name = "Document Management";
+
+    	setUp {
+		task ("Opening the portal and doing the Login") {
+			TestCase.setUpPortalInstance();
+
+			User.firstLoginPG();
+		}
+
+		task ("Creating a Page with a Documents and Media widget") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Documents and Media page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				column = "1",
+				groupName = "Guest",
+				layoutName = "Documents and Media page",
+				widgetName = "Documents and Media");
+				}
+
+        task ("Entering The Documents and Media page") {
+			Navigator.gotoPage(pageName = "Documents and Media page");
+		}
+
+        }
+     	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+                
+            JSONLayout.deletePublicLayout(
+				groupName = "Guest",
+				layoutName = "Documents and Media page");
+
+			DMDocument.tearDownCP();
+			User.logoutPG();
+		}
+	}
+
+    @description = "Can Create a Documents and Media"
+    @priority = "5"
+    test CanCreateDM {
+
+		task ("the user create a Documents and Media (Upload the file)") {
```

Comment:
Oi @Natanaelbenvindo você poderia pensar em uma mensagem como:

Criar um novo Documents and Media